src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java

```diff
@@ -125,13 +125,13 @@ public void run(RunConfiguration config) throws AppEngineException {
   public void stop(StopConfiguration configuration) throws AppEngineException {
     Preconditions.checkNotNull(configuration);
     HttpURLConnection connection = null;
+    String adminHost =
+        configuration.getAdminHost() != null ? configuration.getAdminHost() : DEFAULT_HOST;
```

Comment:
@chanseokoh that is correct. Dev Appserver 1 does not have the concept of admin port. The "stop" listens on the port the app server was launched from.